app/exports/qualifications_export.yml

```diff
@@ -0,0 +1,118 @@
+common_columns:
+  - candidate_id
+  - support_reference
+  - phase
+  - recruitment_cycle_year
+  - choice_status
+  - rejection_reason
+  - course_code
+  - provider_code
+
+custom_columns:
+  gcse_maths_grade:
+    type: string
+    description: GCSE maths grade
```

Comment:
We _could_ list the enum of possible grades here. But they would be repeated a lot and would make the doc harder to read and less useful IMHO. What do you think? Is there a link we can provide to a list? Similar to how we describe `sex` by providing a link to HESA.

Reply:
I suppose we could link to the API's reference data endpoint for GCSE grades: `/api/v1/reference-data/gcse-grades`? We'd need to `skip_before_action :require_valid_api_token!` in the `VendorAPI::ReferenceDataController` but no reason not to do that.

Reply:
Or I guess we could have a page which listed each set in `config/initializers/qualifications`?

Reply:
I think I'd prefer to make a separate page for these details. It would be laid out better and more user friendly, and from what I understand the API only includes data on GCSE and Alevels. If we have a separate page, we could also include the lists of degrees data and other qualifications

Reply:
Prototype idea

![image](https://user-images.githubusercontent.com/33458055/112131943-004e9b80-8bc2-11eb-81f8-f8f503d5600e.png)

Reply:
I really like this 👌

Reply:
+10!